frontend/src/views/Transactions/data.js

```diff
@@ -0,0 +1,295 @@
+export const dummy = {
```

Comment:
I believe it would be outside the scope of whats written in the ticket and would be better tackled when the issuance and transfer data can be implemented ( which I think would make sense when we are doing the Transfer page tasks ). From what I can see, the tasks involved would be:

- getting BCeID accounts up and running ( I think Al or Justin were setting those up )
- being able to write transfer and issuances to those BCeID users ( being tackled with the Transfer page tasks )
- alternatively seeding transfers and issuances to BCeID users

In preparation for the above to be implemented, I have done some scaffolding so it can easily be implemented in the future